server/routes/activitiesMigration/activitiesMigrationController.ts

```diff
@@ -374,6 +374,7 @@ export default class ActivitiesMigrationController {
         startedDate,
       )}) .. datetime(${ActivitiesMigrationController.toISODateTime(endedDate)}))
       | where customDimensions.migrationId startswith '${migrationId}'
+      | where name startswith 'activity-migration'
```

Comment:
For all migration events, don't show the internal events such as splitting by page etc.